src/runtime/virtcontainers/fs_share_linux.go

```diff
@@ -29,6 +29,31 @@ import (
 	"github.com/kata-containers/kata-containers/src/runtime/virtcontainers/utils"
 )
 
+// Splitting Regex pattern:
+// configVolRegex: Regex to match directory structure for k8's volume mounts.
+// Use regex for strict matching instead of strings.Contains
+// match for kubernetes.io~configmap, kubernetes.io~secret, kubernetes.io~projected, kubernetes.io~downward-api
+// as recommended in review comments for PR #7211
```

Comment:
`// as recommended in review comments for PR #7211` -- I'd personally not add this, but as it's I wouldn't remove either. :-)